src/components/ContentSidebar/SkillsSidebar.js

```diff
@@ -196,4 +196,4 @@ class SkillsSidebar extends React.PureComponent<Props, State> {
 }
 
 export { SkillsSidebar as SkillsSidebarComponent };
-export default withErrorBoundary(withAPIContext(SkillsSidebar));
+export default withErrorBoundary(ERROR_TYPE_CONTENT_SIDEBAR)(withAPIContext(SkillsSidebar));
```

Comment:
Should each sidebar have its own type rather than using its parent?

Reply:
This could be helpful as other teams start to inject in the sidebar.